### PR TITLE
improvement: add header to webxdc dev tools element

### PR DESCRIPTION
**Pull request by WofWca**
So that it's clear that it's not a part of the actual app. It confused me a little when I was just starting with webxdc.

### CODE REVIEW

**Comment by r10s** [comment]
i understand the idea of making the two buttons somehow as not being part of the app, however, the current approach takes too much space (it is already before annoying sometimes and you have to resize the window to be able to "click through").

i would make the control smaller, not larger. otherwise it helps one time, and then annoys all other times :)

also, it does not look as good as before.

so, maybe better reword the "Add Peer" so that it is clearer that it is a testing tool? sth. as "+Test  Peer"?

and/or, we can  also add a tooltip

**Comment by WofWca** [comment]
> takes too much space

How about make it move-able and/or hide-able, like `eruda.js`

**Comment by r10s** [comment]
sure, movable, collapsable etc. would be nice as well, but also another effort. and i still think, things should be as small as possible, a large, permanent title would still be annoying most of the time

what about the idea of `+Test Peer` - instead of `Add Peer` - that would make things clear that it is a testing tool and not a part of the final app and solve the initial issue with close-to-zero effort.

File: webxdc.js [hunk 1]
```diff
@@ -213,11 +213,12 @@ window.alterXdcApp = () => {
...
 
   if (window.webxdc.selfName === "device0") {
-    var div = document.createElement("div");
-    div.innerHTML =
+    var root = document.createElement("section");
+    root.innerHTML =
       '<div id="webxdc-panel" style="' +
       styleControlPanel +
       '">' +
+      '<header style="margin-bottom: 0.5em;">webxdc dev tools</header>' +
```
Comment (r10s) [inline webxdc.js:221]:
```suggestion
      '<header style="margin-bottom: 0.5em; font-size:12px;">webxdc dev tools</header>' +
```

**Review by r10s — APPROVED** [review]
i played around a little with font sizes, the title a little smaller seems good enough for now. 

and if we go for `+Peer` the overall size would not be larger at all (any ideas how to make "Clear Storage" smaller?) - but that can be done later as well, no blocker.

Screenshot 2023-09-08 at 13 39 51

**Comment by WofWca** [comment]
> any ideas how to make "Clear Storage" smaller?

"Reset" - and it's also clearer.

**Comment by r10s** [comment]
indeed, "Clear storage" also reloads the page.

so `+Peer` and `Reset` - nice!

**Comment by r10s** [comment]
did a successor: https://github.com/webxdc/hello/pull/55
